safe/gui/tools/help/definitions_help.py

```diff
@@ -742,6 +746,52 @@ def content():
     message.add(table)
 
     ##
+    # Reporting
+    ##
+    _create_section_header(
+        message,
+        table_of_contents,
+        'reporting',
+        tr('Reporting'),
+        heading_level=1)
+
+    message.add(m.Paragraph(tr(
+        'This section of the help documentation is intended for advanced '
+        'users who want to modify reports which are produced by InaSAFE.'
+    )))
+
+    _create_section_header(
+        message,
+        table_of_contents,
+        'reporting',
+        tr('QGIS Expressions'),
+        heading_level=2)
+
+    qgis_expressions = {
+        fct[0]: fct[1] for fct in getmembers(expressions)
```

Comment:
Perhaps you can put some comment here to explain what is fct and why it can be filtered by `QgsExpressionFunction`